pkg/providers/imagefamily/resolver.go

```diff
@@ -37,12 +39,24 @@ const (
 	networkPluginAzure   = "azure"
 	networkPluginKubenet = "kubenet"
 
+	networkPolicyCilium = "cilium"
+
 	// defaultKubernetesMaxPodsAzure is the maximum number of pods to run on a node for Azure CNI Overlay.
 	defaultKubernetesMaxPodsAzure = 250
 	// defaultKubernetesMaxPodsKubenet is the maximum number of pods to run on a node for Kubenet.
 	defaultKubernetesMaxPodsKubenet = 100
 	// defaultKubernetesMaxPods is the maximum number of pods on a node.
 	defaultKubernetesMaxPods = 110
+
+	// AzureCNI VNET Labels
+	vnetDataPlaneLabel      = "kubernetes.azure.com/ebpf-dataplane"
+	vnetNetworkNameLabel    = "kubernetes.azure.com/network-name"
+	vnetSubnetNameLabel     = "kubernetes.azure.com/network-subnet"
+	vnetSubscriptionIDLabel = "kubernetes.azure.com/network-subscription"
+	vnetGUIDLabel           = "kubernetes.azure.com/nodenetwork-vnetguid"
+	vnetPodNetworkTypeLabel = "kubernetes.azure.com/podnetwork-type"
+
+	overlayNetworkType = "overlay"
```

Comment:
It feels like these network constants  (especially azure/kubenet/overlay/cilium) really belong elsewhere ...

Reply:
yes i agree we can either search for them upstream or start an api in agentbaker for node bootstrapping constants. They are needed there too